input/pagecontent/variance.md

```diff
@@ -4,13 +4,5 @@ This implementation guide has no variance (i.e. fully compliant) from AU Base FH
 #### Additionally Profiled Resources
```

Comment:
At this time is makes sense to add in the introductory text to this page to explain what this page does. We would then submit a PR to eReq to incorporate the same text.

Reply:
Added description to the page